PWGHF/HFL/Tasks/taskElectronWeakBoson.cxx

```diff
@@ -147,27 +147,29 @@ struct HfTaskElectronWeakBoson {
       zorroSummary.setObject(zorro.getZorroSummary());
     }
 
+    // add configurable for CCDB path
+    zorro.setBaseCCDBPath(cfgCCDBPath.value);
+
     // define axes you want to use
-    const AxisSpec axisZvtx{400, -20, 20, "Zvtx"};
+    const AxisSpec axisZvtx{40, -20, 20, "Zvtx"};
```

Comment:
Consider to make it configurable to avoid changing the code when you want to modify the number of bins and the range (same for all the axes below). See e.g.
https://github.com/AliceO2Group/O2Physics/blob/2520da892049238a17af1cd415a568e650695978/PWGLF/Tasks/QC/vertexQA.cxx#L80